modules/nf-core/somalier/relate/tests/main.nf.test.snap

```diff
@@ -0,0 +1,286 @@
+{
+    "[ delete_me, []\u00a0], [] -stub": {
+        "content": [
+            {
+                "0": [
+                    [
+                        {
+                            "id": "cohort",
+                            "single_end": false
+                        },
+                        "cohort.html:md5,d41d8cd98f00b204e9800998ecf8427e"
+                    ]
+                ],
+                "1": [
+                    [
+                        {
+                            "id": "cohort",
+                            "single_end": false
+                        },
+                        "cohort.pairs.tsv:md5,d41d8cd98f00b204e9800998ecf8427e"
+                    ]
+                ],
+                "2": [
+                    [
+                        {
+                            "id": "cohort",
+                            "single_end": false
+                        },
+                        "cohort.samples.tsv:md5,d41d8cd98f00b204e9800998ecf8427e"
+                    ]
+                ],
+                "3": [
+                    "versions.yml:md5,59d805a9f89558414535c136c814bea6"
+                ],
+                "html": [
+                    [
+                        {
+                            "id": "cohort",
+                            "single_end": false
+                        },
+                        "cohort.html:md5,d41d8cd98f00b204e9800998ecf8427e"
+                    ]
+                ],
+                "pairs_tsv": [
+                    [
+                        {
+                            "id": "cohort",
+                            "single_end": false
+                        },
+                        "cohort.pairs.tsv:md5,d41d8cd98f00b204e9800998ecf8427e"
+                    ]
+                ],
+                "samples_tsv": [
+                    [
+                        {
+                            "id": "cohort",
+                            "single_end": false
+                        },
+                        "cohort.samples.tsv:md5,d41d8cd98f00b204e9800998ecf8427e"
+                    ]
+                ],
+                "versions": [
+                    "versions.yml:md5,59d805a9f89558414535c136c814bea6"
+                ]
+            }
+        ],
+        "meta": {
+            "nf-test": "0.8.4",
+            "nextflow": "24.04.2"
+        },
+        "timestamp": "2024-06-28T10:28:03.681901901"
+    },
+    "[ delete_me, []\u00a0], []": {
```

Comment:
This is weird, unicode space? (and several more)

Reply:
Something is up with my terminal setup, will fix these.